wradlib/tests/test_georef.py

```diff
@@ -807,6 +808,7 @@ class Data:
             ]
         )
 
+    gc.collect()
```

Comment:
Can you please add a comment here, so that we keep track of why we garbage collect before yield.

Reply:
A comment has been added.